src/interface/storage.thrift

```diff
@@ -218,16 +213,19 @@ struct AddEdgesRequest {
     3: bool overwritable,
 }
 
-struct EdgeKeyRequest {
+struct EdgeKeysRequest {
```

Comment:
Could we use get neighbors directly.  Actually,  "getEdgeKeys" will be deleted in the future. It is meaningless to change it again.